### PR TITLE
fix: use more accessible endpoint for getting driver name in driver install

### PR DESCRIPTION
**Pull request by rossiam**
The endpoint we were using for this is only accessible to the owner of the channel.

## Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **[CONTRIBUTING](../CONTRIBUTING.md)** document
- [x] My code follows the code style of this project (`npm run lint` produces no warnings/errors)
- [ ] Any required documentation has been added
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ OPTIONS
   --language=language    ISO language code or "NONE" to not specify a language. Defaults to the OS locale
 ```
 
-_See code: [src/commands/edge/channels.ts](https://github.com/SmartThingsCommunity/edge-cli-plugin/blob/v0.0.0/src/commands/edge/channels.ts)_
+_See code: [dist/commands/edge/channels.ts](https://github.com/SmartThingsCommunity/edge-cli-plugin/blob/v1.0.4/dist/commands/edge/channels.ts)_
```
Comment (john-u) [inline README.md:69]: If these are using `dist`, the links will 404.

**Review by john-u — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rossiam) [reply README.md:69]: Well, shoot. I'll look into seeing if there is any way to configure oclif-dev. It seems to come from this line in package.json:

```
	"oclif": {
		"commands": "./dist/commands",
```

but if we use `src` there, the packaged (with pkg) binary throws an error when you run an edge command.

Comment (rossiam) [reply README.md:69]: Ok, I think I know how to fix this now. Running `oclif-dev readme --help` says:

> Customize the code URL prefix by setting oclif.repositoryPrefix in package.json.

I'll open an issue for this.

**Pull request merged**